SpatialGDK/Source/SpatialGDKServices/SpatialGDKServices.Build.cs

```diff
@@ -14,7 +14,9 @@ public SpatialGDKServices(ReadOnlyTargetRules Target) : base(Target)
 				"Core",
 				"CoreUObject",
 				"EngineSettings",
-				"UnrealEd"
-			});
+				"UnrealEd",
```

Comment:
I feel like this should be alphabetized.

Reply:
I feel like this is correct but I'm not coding that as it's a *micro-optimization*